### PR TITLE
perf(editor): Improve performance when opening large workflows with node issues

### PR DESCRIPTION
**Pull request by MiloradFilipovic**
This PR should improve performance when opening large workflows that contain node issues (Fixes ADO-1521)

**Background**
Main reason for this poor performance is that our `getCurrentWorkflow()` store getter is unnecessarily heavy but on top of that we are calling it more than we need. This addresses the second part of the issue by changing the following:
- Pausing node issue processing while workflows are loading
- Only getting current workflow once (instead for every node) when calling `refreshNodeIssues`

**Benchmark**
This was tested on a workflow attached to [this Linear ticket](https://linear.app/n8n/issue/ADO-1501/deliveryhero-enterprise-instance-very-slow-loading-workflows) and this fix brings down opening time  from **~1m10s** to **~28s** on my laptop.

**Tests**
- [Latest e2e tests run](https://github.com/n8n-io/n8n/actions/runs/7060874994)

https://community.n8n.io/t/ui-very-slow-with-more-than-100-nodes/8236/14

### CODE REVIEW

File: packages/editor-ui/src/mixins/genericHelpers.ts [hunk 1]
```diff
@@ -29,6 +29,9 @@ export const genericHelpers = defineComponent({
 		readOnlyEnv(): boolean {
 			return this.sourceControlStore.preferences.branchReadOnly;
 		},
+		isLoading(): boolean {
```
Comment (mutdmour) [inline packages/editor-ui/src/mixins/genericHelpers.ts:32]: hope we can get rid of these mixins soon.. maybe this is better to keep in nodeview?

**Review by mutdmour — DISMISSED** [review]
awesome

File: packages/editor-ui/src/mixins/genericHelpers.ts (hunk 1 above)
Comment (MiloradFilipovic) [reply packages/editor-ui/src/mixins/genericHelpers.ts:32]: Yeah me too. Makes sense since it's currently only used there, will update

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.20.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.20.0)

**Pull request merged**